extensions/ui/org.eclipse.smarthome.ui.classic/snippets/group.html

```diff
@@ -1,6 +1,6 @@
 <li class="iGroup">
     <a href="#_%id%" onclick="AsyncLoad('%id%')">
-        <img src="../icon/%category%.%format%" width=29 height=29 />
+        <img src="../icon/%category%?format=%format%&state=%state%" width=29 height=29 />
```

Comment:
Does this correctly work if there is no state (which is probably the normal situation for Groups)?